Packs/CiscoSMA/pack_metadata.json

```
@@ -1,18 +1,22 @@
 {
-    "name": " Cisco SMA ",
-    "description": "Modeling Rules for the Cisco Security Management Appliance logs collector",
+    "name": "CiscoSMA",
+    "description": "The Security Management Appliance (SMA) is used to centralize services from Email Security Appliances (ESAs) and Web Security Appliances (WSAs).",
     "support": "xsoar",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
```

This is a significant change

```suggestion
    "currentVersion": "1.1.0",
```